conanfile.py

```diff
@@ -23,22 +23,27 @@ class Xrpl(ConanFile):
         'unity': [True, False],
     }
 
+    import os
+
+    # Check if we're in HBB release build environment
+    IS_HBB_BUILD = os.getenv('HBB_RELEASE_BUILD') == '1'
```

Comment:
It may be clearer to simply use the variable name `IS_STATIC_BUILD`.

HBB is a tool for static builds.

Reply:
I'll have a quick look at some alternatives while I'm here anyway